packages/dashboard/src/features/compute/pages/ComputePage.tsx

```
@@ -212,20 +212,32 @@ export default function ComputePage() {
+                        </a>
+                      ) : (
+                        <code className="text-foreground font-mono bg-[var(--alpha-8)] px-2 py-0.5 rounded">
+                          {reachableUrl.display}
+                        </code>
+                      )
                     ) : (
                       <span className="text-muted-foreground">Not available</span>
                     )}
+                    {reachableUrl && !reachableUrl.href && (
+                      <p className="text-xs text-muted-foreground mt-1">
+                        Raw TCP service. Connect with your protocol's native client (e.g.{' '}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)